src/components/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+const Footer = () => {
+    return (
+        <>
```

Comment:
As you're returning only one element, not sure that react fragments <></> are needed here.